utils/tests/utils.go

```diff
@@ -476,6 +477,9 @@ func getSubstitutionMap() map[string]string {
 
 // Add timestamp to builds and repositories names
 func AddTimestampToGlobalVars() {
+	if timestampAdded {
```

Comment:
Do you think it is clear enough to someone that doesn't familiar with the original issue ? 
I added the following comment: 
`	// Make sure the global timestamp is added only once even in case of multiple  tests flags`